### PR TITLE
Fixes to acceptMission method

### PR DESCRIPTION
**Pull request by laredotornado**

### CODE REVIEW

File: src/app/model/Mission.ts [hunk 1]
```diff
@@ -154,6 +154,8 @@ class Mission extends BaseModel {
       return [];
     }
     const missions = missionsAvailableForEveryone.docs.map((doc) => doc.data());
+    console.log("all missions:");
+    console.log(missions);
```
Comment (jwu910) [inline src/app/model/Mission.ts:158]: Can we remove unnecessary console logs?

**Comment by jwu910** [comment]
Also you have some errors that should get fixed

```

/home/runner/work/resilience-app/resilience-app/src/app/model/User.test.ts
##[warning]   7:10  warning  'mockBaseRepo' is defined but never used       @typescript-eslint/no-unused-vars
##[warning]  71:9   warning  'mockData' is assigned a value but never used  @typescript-eslint/no-unused-vars

✖ 2 problems (0 errors, 2 warnings)
```

File: src/app/model/Mission.ts (hunk 1 above)
Comment (laredotornado) [reply src/app/model/Mission.ts]: My bad @jwu910 , they're gone.

**Comment by jwu910** [comment]
Is this working against an open issue on github or an airtable ticket?

File: src/app/page/MissionCreate/MissionCreate.jsx [hunk 2]
```diff
@@ -48,7 +48,7 @@ function MakeMission({ history }) {
 
       //create mission object
       const mission = {
-        organisationId: "", // PLACEHOLDER -> user doesn'T have an organisation id yet
+        organisationId: "1", // PLACEHOLDER -> user doesn'T have an organisation id yet
```
Comment (azmainamin) [inline src/app/page/MissionCreate/MissionCreate.jsx:51]: Spelling typo: organization

**Pull request merged**